graphql-client-support/pom.xml

```diff
@@ -40,6 +41,24 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.expediagroup.streamplatform</groupId>
```

Comment:
We should not force the `stream-registry-graphql-client` and `stream-registry-graphql-client-reactor` dependencies on users who simply want the `ObjectNodeTypeAdapter` support.

Reply:
How do you propose we fix that?